### ISSUE TITLE
Fix testing

### ISSUE THREAD

**Issue by elboletaire** [elboletaire/password-strength-meter#13]
Right now, saucelabs browsers testing is failing due to an unknown reason.

I need to check this further and try to fix it, or search for an alternative to saucelabs.

**Comment by elboletaire** [comment]
It's weird.. I'm thinking in removing the saucelabs integration, although it's cool... it gives random errors. When I created this issue it was just failing; taking a look to the saucelabs page didn't show any record and taking a look at travis builds basically showed you all the saucelabs stuff crashing.

Now it works, but some browsers fail... maybe I'll just remove those failing browsers.. dunno

**Comment by jhoover4** [comment]
Whats failing? All the tests run fine for me.

**Comment by elboletaire** [comment]
Sorry didn't see this @jhoover4. The problem was with saucelabs integration, which was working from time to time. I've ended up replacing all the karma + istanbul + jasmine (+ the jasmine jquery plugin and other dependencies) stack with just Jest.

About the saucelabs integration... I've ended up removing it. They only store data for a few days, which makes the badge to fail + there were tests failing under some of the environments, when they shouldn't; so at the end I prefer not to use it.
